src/main/java/appeng/block/misc/BlockInterface.java

```diff
@@ -52,6 +57,26 @@ public BlockInterface()
 	}
 
 	@Override
+	protected IProperty[] getAEStates()
+	{
+		return new IProperty[] { AE_BLOCK_FORWARD, AE_BLOCK_UP, OMNIDIRECTIONAL };
```

Comment:
Well we're talking block states here. If you extend the property container, you'll most likely have to extend the block variants as well unless you also touch the state mapper. 
In that cleanup we might also come to the conclusion that FORWARD/UP should be unlisted properties anyway and remove our state mapper that currently hides them. Otherwise it's always a bit unclear which properties actually affect the variant-space of a block and which don't. When I started contributing this was one of the things that confused me: That there were no variants required for all forward/up combinations, since I didn't know about the custom state mapper.